node/libs/crypto/src/bls12_381/mod.rs

```diff
@@ -1,8 +1,8 @@
-//! BLS12-381 signature scheme.
+//! This module implements the BLS signature over the BLS12_381 curve.
```

Comment:
While you are at it, please add links to the respective RFCs which you are following to implement your changes.

Reply:
added